plugins/admin/README.adoc

```diff
@@ -114,6 +114,56 @@ Global Flags:
 
 Use "kn admin autoscaling [command] --help" for more information about a command.
 
+----
+
+#### `kn admin profiling`
+
+----
+Enable Knative components profiling and download profile data
```

Comment:
@lanceliuu @chaozbj 
I think we need to specify the scope here. `profiling` here works for Knative Serving but not Eventing. Knative components are too generic. Maybe change to Knative Serving profiling

Reply:
Maybe we can consider to support Knative Eventing as well?

Reply:
@lanceliuu yes, we'll support eventing user scenarios in future, and may need to separate serving and eventing like 
`kn admin serving profiling` and `kn admin eventing profiling`. I think so far this PR is focusing on Serving profiling.

Reply:
@zhanggbj Should we change the command scope to ```kn admin serving profiling``` right now? Since we will support eventing profiling, we always need to do.

Reply:
Talked with @zhanggbj, we will keep using ```kn admin profiling``` now, and will change it when we introduce eventing related commands.